KlingonAssistant/data/mem-05-H.xml

```diff
@@ -925,7 +925,7 @@ A physical path would be a {taw:n}.</column>
       <column name="entry_name">Heghba'</column>
       <column name="part_of_speech">n:alt,fic</column>
       <column name="definition">{Heghbat:n}</column>
-      <column name="definition_de"></column>
+      <column name="definition_de">{Heghbat:n}</column>
```

Comment:
For ones like these, you don't have to put the German definition. If it's empty, it defaults to the English. That's better anyway: it takes less space, and if the definition needs to changed, then it needs to be changed only once rather than twice.